src/integrations/admin/configuration-workout-integration.php

```diff
@@ -125,7 +135,7 @@ public function enqueue_assets() {
 		$this->admin_asset_manager->localize_script( 'indexation', 'yoastIndexingData', $data );
 
 		$social_profiles        = $this->get_social_profiles();
-		$person_social_profiles = $this->get_person_social_profiles();
+		$person_social_profiles = ( $this->is_company_or_person() === 'person' ) ? $this->social_profiles_helper->get_person_social_profiles( $this->get_person_id() ) : \array_combine( $this->social_profiles_helper->get_person_social_profiles_fields(), \array_fill( 0, count( $this->social_profiles_helper->get_person_social_profiles_fields() ), '' ) );
```

Comment:
I find this really convoluted. I'd go with a good old `if() {}`, makes it also more testable.